scene/gui/code_edit.cpp

```diff
@@ -3193,11 +3208,16 @@ void CodeEdit::_filter_code_completion_candidates_impl() {
 			if (theme_cache.font.is_valid()) {
 				max_width = MAX(max_width, theme_cache.font->get_string_size(option.display, HORIZONTAL_ALIGNMENT_LEFT, -1, theme_cache.font_size).width + offset);
 			}
-			code_completion_options.push_back(option);
+			code_completion_options_new.push_back(option);
 		}
 
+		code_completion_options_new.sort_custom<CodeCompletionOptionCompare>();
```

Comment:
No need to sort here, as it's overridden allowing the user set it as they see fit